typescript/sdk/src/consts/multisigIsm.ts

```diff
@@ -522,7 +522,7 @@ export const defaultMultisigConfigs: ChainMap<MultisigConfig> = {
     validators: ['0xfa33391ee38597cbeef72ccde8c9e13e01e78521'],
   },
 
-  solana: {
+  solanamainnet: {
```

Comment:
> /home/runner/work/hyperlane-monorepo/hyperlane-monorepo/typescript/sdk/src/consts/multisigIsm.ts
  530:3  error  Expected object keys to be in ascending order. 'solanadevnet' should be before 'solanamainnet'  sort-keys